test/cosmoz-treenode-button-view.test.ts

```diff
@@ -0,0 +1,112 @@
+import { assert, fixture, html } from '@open-wc/testing';
+import { DefaultTree } from '@neovici/cosmoz-tree/cosmoz-default-tree';
+import rtlData from './data/rtlTest';
+import '../src/cosmoz-treenode-button-view';
+
+suite('rtl', () => {
+	let treeButton: any;
+
+	suiteSetup(async () => {
+		treeButton = await fixture(
+			html`<cosmoz-treenode-button-view
+				.tree=${new DefaultTree(rtlData)}
+			></cosmoz-treenode-button-view>`,
+		);
+	});
+
+	test('button renders selected path', (done) => {
+		treeButton.nodePath = '1.2.3';
+		setTimeout(() => {
+			const buttonLabel = treeButton.shadowRoot
+				.querySelector('.actions__open-dialog')
+				.textContent.trim();
+			assert.isTrue(buttonLabel.endsWith('1 / 2 / 3'));
+			done();
+		}, 500);
+	});
+
+	test('opens dialog on button click', async () => {
+		const openButton = treeButton.shadowRoot.querySelector(
+			'.actions__open-dialog',
+		);
+		openButton.click();
+
+		await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
```suggestion
		await nextFrame();
```

`import { assert, fixture, html, nextFrame } from '@open-wc/testing';`